### DISCUSSION TITLE
issue 反馈模版里的重现链接，能否增加 playground 地址支持

### DISCUSSION THREAD

**Discussion by hooray** [element-plus/element-plus#5809] [Ideas]
虽然提供了下面这三个模版，但实际反馈过程还是比较繁琐，还是需要在模版的基础上编写复现代码。
[jsfiddle](https://jsfiddle.net/iamkun/ods20k7t/4/)
[codepen](https://codepen.io/iamkun/pen/YzWMaVr)
[codesandbox](https://codesandbox.io/s/element-plus-spa-ms973?file=/src/App.vue)

而一些小问题在使用playground测试的时候发现了，把playground地址复制过来却提示链接不正确，希望能增加，或者去掉复现链接的输入验证
![image](https://user-images.githubusercontent.com/1574903/152722661-6cef97a6-c72a-4e37-8b27-3f2ee5e1a47b.png)

**Comment by emojiiii** [comment]
这个问题已经在计划中，感谢你的反馈！

**Comment by sxzz** [comment]
你可以先随便提交一个（三个模版的任意一个），然后跳转到 GitHub 编辑器的时候改改。新的 issue helper 已经在计划中了。
